### DISCUSSION TITLE
关于 mongo 配置参数的建议

### DISCUSSION THREAD

**Discussion by aceyin** [cloudwu/skynet#1863] [General]
目前skynet 里面 mongo 组件有几个配置参数的名字，和 mongo 官方的配置参数名字不相同  
例如:  
>  `authdb` 在 mongo 官方的参数应该是 `authSource`  
>  `authmod` 看起来应该是官方的 `authMechanism`?    
(参考这个链接: https://www.mongodb.com/docs/manual/reference/connection-string/)

建议 skynet 这边也和和官方的参数名保持同步

**Comment by cloudwu** [comment]
我读了文档，但还是不确定 authdb 是不是指的 `authSource` 。`authMechanism` 后面的参数也不确定格式。

我对 mongo 不熟悉，目前的 driver 有很多命名问题，我想不是简单的修改一两个位置能统一的。另外，还有兼容问题不好解决。
